bower.json

```diff
@@ -2,7 +2,7 @@
     "name": "adapt-hotgrid",
     "repository": "git://github.com/cgkineo/adapt-hotgrid",
     "framework": "^2.0.0",
-    "version": "2.0.0",
+    "version": "3.0.0",
```

Comment:
Amended version number and updated Limitation section in the ReadMe with potential problems from upgrading and solutions.